lib/kitchen/directions/bake_index/v1.xhtml.erb

```diff
@@ -1,9 +1,9 @@
-<div class="os-eob os-index-container " data-type="composite-page" data-uuid-key="index">
+<div class="os-eob <%= @container_class %> " data-type="composite-page" data-uuid-key="<%= @uuid_key %>">
```

Comment:
the space after the interpolated `container_class` tag seems to be creating trailing whitespace in the classes; might be a good idea to clean that up